tools/changelogger/src/render.rs

```diff
@@ -489,20 +503,23 @@ module = "aws-sdk-s3"
 version = "0.14.0"
 kind = "Feature"
 message = "Some new API to do X"
+references = ["smithy-rs#199"]
```

Comment:
`[[aws-sdk-model]]` entries shouldn't have references. Looks like there are about four of these that should be removed.